examples/authorize-beta/src/main/java/com/dropbox/core/examples/authorize_beta/IncrementalScopeAuthorize.java

```diff
@@ -0,0 +1,62 @@
+package src.main.java.com.dropbox.core.examples.authorize_beta;
+
+import com.dropbox.core.DbxAppInfo;
+import com.dropbox.core.DbxAuthFinish;
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.DbxWebAuth;
+import com.dropbox.core.IncludeGrantedScopes;
+import com.dropbox.core.TokenAccessType;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Use include_granted_scopes when you ask for a small amount of permission at beginning, and
+ * then incrementally ask for more permission when user hit certain feature.
+ * The token contains all previously granted scopes.
+ */
+public class IncrementalScopeAuthorize {
+    public DbxAuthFinish authorize(DbxAppInfo appInfo) throws IOException {
```

Comment:
Havent had a chance to look at the other examples, but wouldn't an example app just run under main?